### PR TITLE
Left overs bug fix

### PR DESCRIPTION
**Pull request by ZappoMan**
Sometimes laser beams and flashlight lights are left around. This PR improves that situation by keeping some additional erase entities history and sending it to viewers over a longer time window.

This doesn't address the core issue (which I have still not found yet) where for some reason the entity IDs are mixed up when being sent from the server to some viewers.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 3316b63bf6e8efb1c9a494dd5ce027983b76c026 available at: https://gist.github.com/a650858209661a5dd018<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6366.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6366.dmg

File: assignment-client/src/entities/EntityNodeData.h [hunk 1]
```diff
@@ -18,9 +18,7 @@
 
 class EntityNodeData : public OctreeQueryNode {
 public:
-    EntityNodeData() :
-        OctreeQueryNode(),
-        _lastDeletedEntitiesSentAt(0) { }
+    EntityNodeData() : OctreeQueryNode() { }
```
Comment (Atlante45) [inline assignment-client/src/entities/EntityNodeData.h]: Pretty sure you can just remove the actor altogether here.

File: libraries/entities/src/EntityTree.cpp [hunk 2]
```diff
@@ -384,16 +384,17 @@ void EntityTree::deleteEntities(QSet<EntityItemID> entityIDs, bool force, bool i
...
     foreach(const EntityToDeleteDetails& details, entities) {
         EntityItemPointer theEntity = details.entity;
 
         if (getIsServer()) {
             // set up the deleted entities ID
-            quint64 deletedAt = usecTimestampNow();
-            _recentlyDeletedEntitiesLock.lockForWrite();
-            _recentlyDeletedEntityItemIDs.insert(deletedAt, theEntity->getEntityItemID());
-            _recentlyDeletedEntitiesLock.unlock();
+            {
```
Comment (Atlante45) [inline libraries/entities/src/EntityTree.cpp]: Not blocking but those curly braces are redundant.

Comment (ZappoMan) [inline libraries/entities/src/EntityTree.cpp]: duh - missed that because I had moved deletedAt outside later

**Comment by hifi-gustavo** [comment]
Build console output for commit: 866116d2855baf69b25777cde78cbafe4c66741a available at: https://gist.github.com/a650858209661a5dd018<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6366.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6366.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5576cf188e68b9c2ad91715c87d1dc386a310b9a available at: https://gist.github.com/a650858209661a5dd018<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6366.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6366.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 040bae60142ef9ed611510382cae777f64d77450 available at: https://gist.github.com/a650858209661a5dd018<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6366.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6366.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: bdfe304f7aee9f8b6a8b3e6969b8146cea690aaa available at: https://gist.github.com/a650858209661a5dd018<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6366.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6366.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2f903a9513f43a7e5da301a756e4eccf082359bb available at: https://gist.github.com/a650858209661a5dd018<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6366.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6366.dmg

**Pull request merged**